### ISSUE TITLE
Optionally disable the "Close modal" button

### ISSUE THREAD

**Issue by liamaharon** [blocknative/assist#270]
It sometimes makes sense for the completion of onboarding to be compulsory, where the dev doesn't want to handle a 'onboard failed' scenario. This could be achieved by adding the option to disable to top right "close modal" button from the onboard modals.

**Comment by cmeisl** [comment]
@liamaharon Do we have an actual use case here from any of our integrations?

**Comment by liamaharon** [comment]
@cmeisl yes, on some pages space-miners and now mintable require the user to be onboarded before the page loads. if the user fails to onboard, the app falls into an error state
